apps/mobile/app/(drawer)/(tabs)/_layout.tsx

```diff
@@ -1,13 +1,15 @@
 import { Tabs } from 'expo-router'
-import { House, BookmarkSimple } from 'phosphor-react-native';
+import { House, BookmarkSimple } from 'phosphor-react-native'
+
+import { TabsHeader } from '../../components/TabsHeader'
```

Comment:
Podemos usar a config de path alias?
```suggestion
import { TabsHeader } from '@/components/TabsHeader'
```

Reply:
eu tentei também utilizar na #79 e quebrou várias coisas do expo. Será que precisamos fazer alguma configuração específica? Minha suposição é alguma coisa com o expo-router

Reply:
@diego3g  sobre o alias, não consegui resolver. 😕
Ao que parece o babel resolver não está entendendo a pasta `app` como diretório root, talvez o @rodrigorgtic saiba como resolver isso e possamos abrir uma issue para corrigir esse path mapping, o que achas?

Reply:
@LuizHAP provavelmente seja algo do próprio plugin do module resolver mesmo. 😕

Reply:
Acredito que algumas incosistências do module resolver sejam por causa que quando o ele foi configurado foi levado em conta que a raiz da aplicação seria o `./src`. No entanto, algumas dependências estão sendo criadas simultaneamentes na `app` e `src`. Até isso ser padroniziado, imagino que seja melhor seguir como o @LuizHAP fez abaixo:

https://github.com/diego3g/rsxp-2023/blob/main/apps/mobile/babel.config.js#L21-L24